docs/blog/newsletter-april-2022.rst

```diff
@@ -0,0 +1,137 @@
+#########################################
+
+Hey everyone! It's Beth and the newsletter team, back with another edition full of news and views from the docs world.
+
+2022 is a big year for Write the Docs: it will be our 10th year of conferences in North America. To celebrate, we've added a video of an O'Reilly Interview of Eric and Marcia Riefer Johnston from 2015 to our "Origin Story" page. `Watch and enjoy </origin-story/#where-we-are-now>`__!
+
+And if you fancy joining us for that 10th NA conference, you might like to know that we've just announced our `fabulous speakers </conf/portland/2022/news/announcing-speakers/>`__. Lots to get excited about there!
+
+To finish, a bit of news from your editor. After four lovely years and 35 (!!) issues, it's time for me to hand over stewardship of the newsletter to someone new. This is such a great opportunity to give back to the WTD community; so if you think it might be for you, take a look at `this blog post about the role </blog/newsletter-editor/>`__ and get in touch.
+
+----------------------------------
+Where should support content live?
+----------------------------------
+
+When it comes to making support content (like a knowledge base) discoverable and offering a consistent experience for users, what is the best approach? Should you keep support content with the rest of your documentation? Or should support content have its own separate site?
+
+Some recommended keeping everything together, for example on the same platform. A single access point avoids authors working in silos, and helps to build a "single source of truth" - to avoid duplicated work or contradictory content. If you maintain support content as a documentarian, keeping it with the tech docs facilitates revising content to improve discoverability, rewriting unclear passages, and updating to match style guidelines.
+
+Folks who leaned toward separation pointed out that support content and technical docs are very different and may not fit neatly into one information architecture. Tools for presenting support content can look more like news websites, with links to new and trending content, but without a hierarchical navigation structure for all available content. And if you want integrate support content into a ticketing platform to help users with issues, such platforms usually aren't optimised for docs.
+
+Support docs are often written by non-documentarians like support engineers, who write content on the fly in response to real help requests. If a tech writer doesn't have time to edit everything, it's an advantage to keep support content separate from more polished technical docs. Content that quickly solves a problem is valuable even if it isn't perfectly written; it doesn't make sense to delay publication until it can get writerly attention. For one more perspective, check out `Product Documentation vs. Knowledge Base <https://dzone.com/articles/product-documentation-vs-knowledge-base-1>`_ by Jakub Vul.
```

Comment:
The folks **are** documentarians, just not tech writers.

Reply:
And we switched from "folks said X" to "X is true"?

Reply:
Yeah, this is dodgy in both respects, I hadn't noticed. Fixed.